modules/e2ee/Worker.ts

```diff
@@ -65,25 +99,33 @@ onmessage = event => {
         }
     } else if (operation === 'encode' || operation === 'decode') {
         const { readableStream, writableStream, participantId } = event.data;
-        const context = getParticipantContext(participantId);
 
-        handleTransform(context, operation, readableStream, writableStream);
+        if (readableStream && writableStream && participantId) {
```

Comment:
This is too defensive, throw an exception if they are not there, as that would be a prigramming error.